src/docker/manifest.json

```diff
@@ -3,13 +3,27 @@
     "version": "1.0.0",
     "package": {
         "version": "1.0.0",
-        "depends": [],
-        "name": "ucentral_client"
+        "depends": [
+            {
+                "name": "gnmi"
+            },
+            {
+                "name": "mgmt-framework"         
+            },
+            {
+                "name": "swss"
+            },
+            {
+                "name": "syncd"
+            }
+        ],
+        "name": "ucentral_client",
+        "description": "SONiC uCentral client package"
     },
     "service": {
         "name": "ucentral_client",
         "requires": [],
-        "after": [],
+        "after": ["gnmi", "mgmt-framework", "swss", "syncd"],
```

Comment:
BRCM Sonic doesn't seem to be having explicit GNMI service;
Actually, i think we should try and separate manifests for each different target platform, and we should not try to fit a single one for all supported platforms;